### PR TITLE
executor: migrate test-infra to testify for select_into_test.go

### LINKED ISSUE #28607: migrate test-infra to testify for `select_into_test.go`

### PR DESCRIPTION
**Pull request by feitian124**
### What problem does this PR solve?

Issue Number: close #28607 

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by feitian124** [comment]
@hawkingrei @tisonkun  could you help review, thanks.

File: executor/select_into_test.go [hunk 1]
```diff
@@ -19,73 +19,77 @@ import (
...
 func randomSelectFilePath(testName string) string {
 	return filepath.Join(os.TempDir(), fmt.Sprintf("select-into-%v-%v.data", testName, time.Now().Nanosecond()))
 }
 
-func (s *testSuite1) TestSelectIntoFileExists(c *C) {
+func TestSelectIntoFileExists(t *testing.T) {
 	outfile := randomSelectFilePath("TestSelectIntoFileExists")
 	defer func() {
-		c.Assert(os.Remove(outfile), IsNil)
+		require.Nil(t, os.Remove(outfile))
```
Comment (hawkingrei) [inline executor/select_into_test.go:46]: ```os.Remove``` return error. so it should use ```require.NoError```.

ditto other.

Comment (feitian124) [reply executor/select_into_test.go]: ok fixed

**Review by hawkingrei — APPROVED** [review]

**Review by tisonkun — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**